docs/docker/docker-compose-browserless.yaml

```diff
@@ -22,10 +21,11 @@ services:
     command: [ "next", "start", "-p", "3000" ]
     depends_on:
       - browserless
+    restart: unless-stopped
 
   browserless:
+  container_name: browserless
     image: browserless/chrome:latest
-    ports:
-      - "9222:3000"  # Map host's port 9222 to container's port 3000
```

Comment:
Is this needed? I see it's removed, but it could be important?

Reply:
Note: this does not seem to be needed anymore indeed. do you know if it's safe to remove it for everyone?